README.md

```diff
@@ -1,43 +1,110 @@
-<h1 align="center">OSTIS example app</h1>
+<h1 align="center">ostis-example-app</h1>
 
 [![license](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
 
 ## About
 
-OSTIS example app is an ostis-system based on [**ostis-web-platform**](https://github.com/ostis-ai/ostis-web-platform) and designed with [OSTIS Technology](https://github.com/ostis-ai).
-
-0.9.0 version allows communication with the knowledge base via [JSON-based Websocket protocol](http://ostis-dev.github.io/sc-machine/http/websocket/).
-
+ostis-example-app is an ostis-system based on [**ostis-web-platform**](https://github.com/ostis-ai/ostis-web-platform) and designed with [OSTIS Technology](https://github.com/ostis-ai).
 
 ## Installation
 
-For Ubuntu/Debian based distros:
+### Clone repository
+
 ```sh
-git clone https://github.com/ostis-apps/ostis-example-app.git
+git clone -b release/0.10.0 --recursive https://github.com/NikitaZotov/ostis-example-app.git
 cd ostis-example-app
-git checkout prerelease-0.10.0
-./scripts/install.sh
+```
+
+### Install pipx
+
+To install pipx, use this guide: https://pipx.pypa.io/stable/installation/.
+
+### Install Conan
+
+```sh
+# Use pipx to install conan if not already installed
+pipx install conan
+pipx ensurepath
+```
+
+### Relaunch shell
+
+```sh
+exec $SHELL
+```
+
+### Install sc-machine libraries
+
+```sh
+conan remote add ostis-ai https://conan.ostis.net/artifactory/api/conan/ostis-ai-sc-machine
+conan profile detect && conan install . --build=missing
```

Comment:
```suggestion
conan profile detect
conan install . --build=missing
```
Because I run with && and got error for the first command and the second command didn't run
![Uploading image.png…]()